gbm-cli/cmd/release/status.go

```diff
@@ -39,7 +40,7 @@ var StatusCmd = &cobra.Command{
 			// Check to see if the release has been published
 			rel, _ := release.GetGbmRelease(version)
 
-			if (rel != gh.Release{}) {
+			if (!reflect.DeepEqual(rel, gh.Release{})) {
```

Comment:
This is the only possible regression since the simple equality won't work with array of `ReleaseAsset` added to the `Release` struct. This can be verified by running a status check on a published release e.g.
```
go run main.go release status 1.114.1
```